drivers/eeprom/eeprom_at2x.c

```diff
@@ -260,7 +267,7 @@ static int eeprom_at24_read(const struct device *dev, off_t offset, void *buf,
 	timeout = k_uptime_get() + config->timeout;
 	while (1) {
 		int64_t now = k_uptime_get();
-		err = i2c_write_read(config->bus_dev, bus_addr,
+		err = i2c_write_read(config->bus.i2c.bus, bus_addr,
```

Comment:
Could we use `i2c_write_read_dt()` here?

Reply:
No, because `bus_addr` is not the raw reg address of the node, it is derived in `eeprom_at24_translate_offset`